node/impl/eth/events.go

```diff
@@ -609,6 +578,10 @@ func ethTxHashFromMessageCid(ctx context.Context, c cid.Cid, cs ChainStore) (eth
 // * "block" instead of "height"
 // * strings that can have "latest" and "earliest" and nil
 // * hex strings for actual heights
+//
+// Note: Ethereum supports more than "latest" and "earliest" (e.g. "finalized" and "safe"), but
+// currently we do not. The use-case isn't as clear for this but it's possible to add in the future
+// if needed.
```

Comment:
I decided not to bother for now, maybe if we're ever asked, but it's worth noting the discrepancy